### DISCUSSION TITLE
How to Extend Webhook Timeout

### DISCUSSION THREAD

**Discussion by absolutezack** [postalserver/postal#2647] [Help with using Postal]
Hi,

I am experiencing an issue with my Postal server, where it's failing to deliver webhook requests to Pipedream. I'm receiving a response body of 'Timed out after 5s.' I believe this is due to a network issue. Is there a way to extend the timeout to more than 5 seconds?

Thanks.

**Comment by willpower232** [comment]
It doesn't look like that is configurable currently but if you wanted to add a couple of lines of code and make your own docker image, you could probably sort it out

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/app/models/webhook_request.rb#L57
